### PR TITLE
fix: remove xl breakpoint on /creators/slug

### PR DESCRIPTION
**Pull request by rotimi-best**
Fixes: https://github.com/acekyd/made-in-nigeria/issues/300

## Before this fix
image

## After the fix
image

### CODE REVIEW

**Comment by kaf-lamed-beyt** [comment]
Hi @rotimi-best, thank you for this PR. 

I tried changing the HEAD of your PR to `web-dev` and it shows many commits to be merged, which is understandable because you didn't make your change from the `web-dev` branch. 

Kindly open another PR from the web-dev branch. I mean... make your changes from the web-dev branch in your forked version and open a new PR.

**Comment by rotimi-best** [comment]
@kaf-lamed-beyt oh so the HEAD of this project isn't master?

Alright, will change that for all my PRs 👍🏽😊

File: app/components/Blog/BlogPost.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const BlogPost = ({
         flexFlow="column"
         justifyContent="center"
         mt={{ base: "-8rem", md: "-8rem", lg: "-20rem" }}
-        px={{ base: ".5rem", md: "8rem", lg: "12rem", xl: "32rem" }}
+        px={{ base: ".5rem", md: "8rem", lg: "12rem" }}
```
Comment (kaf-lamed-beyt) [inline app/components/Blog/BlogPost.tsx:42]: And the issue with removing the `xl` breakpoint from here now makes the content too "wide"-ish on screens with the `xl` breakpoint. See 👇🏼 

image

Folks would have to be struggling to read content from left to right. It is better if the content is in the center and it catches their attention immediately. 

A `xl` value of `22rem` should suffice, perhaps. In your next PR. But, feel free to play with the values until you get one that is suitable.

Comment (rotimi-best) [reply app/components/Blog/BlogPost.tsx:42]: can't we just add a max-width to the content?

**Comment by kaf-lamed-beyt** [comment]
> @kaf-lamed-beyt oh so the HEAD of this project isn't master?
> 
> Alright, will change that for all my PRs 👍🏽😊

No, it is. 

We recently just merged all the branches into master. All website-related changes went into the web-dev branch previously. It is just a way for us to have uniformity, and perhaps see the changes in staging before merging. 

Sorry for the trouble.

**Comment by rotimi-best** [comment]
> > @kaf-lamed-beyt oh so the HEAD of this project isn't master?
> > Alright, will change that for all my PRs 👍🏽😊
> 
> No, it is.
> 
> We recently just merged all the branches into master. All website-related changes went into the web-dev branch previously. It is just a way for us to have uniformity, and perhaps see the changes in staging before merging.
> 
> Sorry for the trouble.

ahh I see. so its like a staging branch. Got it.

Maybe eventually we will need to add more dev onboarding related info into the README and move the content of the projects somewhere else or to the bottom.

I noticed the /projects route parses the README so we'd have to decide where to mke the change that won't break that endpoint.

**Comment by kaf-lamed-beyt** [comment]
Yes. the Contributing guideline needs to be updated. We'll get to that.

**Comment by kaf-lamed-beyt** [comment]
> > @kaf-lamed-beyt oh so the HEAD of this project isn't master?
> > Alright, will change that for all my PRs 👍🏽😊
> 
> No, it is.
> 
> We recently just merged all the branches into master. All website-related changes went into the web-dev branch previously. It is just a way for us to have uniformity, and perhaps see the changes in staging before merging.
> 
> Sorry for the trouble.

@rotimi-best, kindly hold on with the new PR. I'll ping you immediately the branches thingy I talked about is stable. 🙏🏼

**Comment by rotimi-best** [comment]
> > > @kaf-lamed-beyt oh so the HEAD of this project isn't master?
> 
> > > Alright, will change that for all my PRs 👍🏽😊
> 
> > 
> 
> > No, it is.
> 
> > 
> 
> > We recently just merged all the branches into master. All website-related changes went into the web-dev branch previously. It is just a way for us to have uniformity, and perhaps see the changes in staging before merging.
> 
> > 
> 
> > Sorry for the trouble.
> 
> 
> 
> @rotimi-best, kindly hold on with the new PR. I'll ping you immediately the branches thingy I talked about is stable. 🙏🏼 

Okay sounds good. 👍🏽

**Comment by rotimi-best** [comment]
@kaf-lamed-beyt Created a new PR with your suggestion and it points to `dev` branch

https://github.com/acekyd/made-in-nigeria/pull/309

**Comment by kaf-lamed-beyt** [comment]
Ayy! I'll take a look now

**Pull request closed without merging**